<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogSettings</class>
 <widget class="QDialog" name="dialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabSettings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabConnection">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Server port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinPort">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>65536</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number of threads</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinThreads">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Connection live timeout</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinConnectionLiveTimeout">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Connection on screen timeout</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinConnectionOnScreenTimeout">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUsers">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableUsers"/>
       </item>
       <item>
        <widget class="QTableWidget" name="tableUserTime">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rowCount">
          <number>7</number>
         </property>
         <property name="columnCount">
          <number>48</number>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
